packages/devextreme/testing/testcafe/tests/editors/slider/slider.ts

```diff
@@ -11,6 +11,8 @@ test('Slider appearance', async (t) => {
 
   await testScreenshot(t, takeScreenshot, 'slider-appearance.png', { element: '#container' });
 
+  await testScreenshot(t, takeScreenshot, 'slider-appearance.png', { element: '#container', theme: `${getFullThemeName().replace('light', 'dark')}` });
```

Comment:
Should we use string literals in this case `${getFullThemeName().replace('light', 'dark')}`?

Reply:
I think it should be string literals. I also studied other implementations for Dark Themed Screenshots in the neighboring files and it's implemented like that.

Reply:
I think there's a typo, and it would be better not to wrap it in `${}`.